Split `mithril-common` phase 2: extract Mithril test http server (#2594)
input-output-hk/mithril: internal/tests/mithril-test-http-server/README.md

# Mithril-test-http-server

> This crate provides an http server specific for tests that use the same `warp` library as the `mithril-aggregator`.

Reviewer: ```suggestion
This crate provides a test http server used by the Mithril nodes`.
```
